process/block/shardblock.go

```diff
@@ -100,6 +102,7 @@ func NewShardProcessor(arguments ArgShardProcessor) (*shardProcessor, error) {
 	headersPool.RegisterHandler(sp.receivedMetaBlock)
 
 	sp.metaBlockFinality = process.BlockFinality
+	sp.userStatePruningQueue = queue.NewHashQueue(arguments.UserStatePruningQueueSize)
```

Comment:
Can be left as it is but these kinds of constructs will make the unit tests writing more complex.